### PR TITLE
[mission-control] Add support secrets for Elastic search

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:
[mission-control] Add support secrets for Elastic search

### CODE REVIEW

File: stable/mission-control/templates/jfmc-setup-scripts.yaml [hunk 1]
```diff
@@ -45,8 +45,8 @@ data:
         touch  ${JFMC_PROPERTIES} || (  echo "unable to create ${JFMC_PROPERTIES} file" && exit 1 )
     fi
 
-    addProperty "elastic.username" "${ELASTIC_SEARCH_USERNAME}" ${JFMC_PROPERTIES} && \
-    addProperty "elastic.password" "${ELASTIC_SEARCH_PASSWORD}" ${JFMC_PROPERTIES} && \
+    addProperty "elastic.username" "${JF_SHARED_ELASTICSEARCH_USERNAME}" ${JFMC_PROPERTIES} && \
```
Comment (danielezer) [inline stable/mission-control/templates/jfmc-setup-scripts.yaml:48]: Is this still needed? I think JFMC can use JF_SHARED_ELASTICSEARCH_PASSWORD directly

Comment (chukka) [reply stable/mission-control/templates/jfmc-setup-scripts.yaml:48]: Yes, Not needed,  Removed  :)

File: stable/mission-control/templates/elasticsearch-secrets.yaml [hunk 2]
```diff
@@ -9,4 +10,13 @@ metadata:
     release: {{ .Release.Name }}
 type: Opaque
 data:
-  password:  {{ required "A valid .Values.elasticsearch.password entry required!" .Values.elasticsearch.password | b64enc | quote }}
+{{- if .Values.elasticsearch.username }}
+  username:  {{ .Values.elasticsearch.username | b64enc | quote }}
```
Comment (danielezer) [inline stable/mission-control/templates/elasticsearch-secrets.yaml:14]: What happens when there's no username and password?

Comment (chukka) [reply stable/mission-control/templates/elasticsearch-secrets.yaml:14]: @danielezer  we have default user/password  set in values.yaml

**Pull request merged**